### PR TITLE
Updated image and some text to match updated UI

### PR DESCRIPTION
**Pull request by paperclypse**
There was a bit of a gap between the UI and our docs.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/use-incident-intelligence.mdx [hunk 1]
```diff
@@ -56,13 +56,13 @@ If you hover over an impacted entity application, you’ll notice a few calls to
...
-## Impacted entities map 
+## Impacted entities issue map 
 
-![A screenshot ofthe issue entity map view.](./images/issue_entity_map.png "The issue entity map view.")
+![A screenshot of the issue entity map view.](./images/issue_entity_map.png "The issue entity map view.")
 
 <figcaption>Hover over an entity to see more information about it.</figcaption>
 
-An impacted entities map is available for any issue involving two or more entities. The [map](/docs/new-relic-one/use-new-relic-one/ui-data/maps-in-context/) shows not only the affected entities, but also the services and resources directly related to those entities.
+In the **Impacted entities** section, an issue map is available for any issue involving two or more entities. The [map](/docs/new-relic-one/use-new-relic-one/ui-data/maps-in-context/) shows not only the affected entities, but also the services and resources directly related to those entities.
```
Comment (kaileyhaynes) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/use-incident-intelligence.mdx:65]: link works great!

**Review by kaileyhaynes — CHANGES_REQUESTED** [review]

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/use-incident-intelligence.mdx [hunk 2]
```diff
@@ -56,13 +56,13 @@ If you hover over an impacted entity application, you’ll notice a few calls to
 * **Anomaly overview** will open the application's anomalies page. This is only available for applications that are set up for Proactive Detection.
 * There are two types of **deployment events**: deployments and related deployments. Click **[Show all deployments](/whats-new/applied-intelligence-deployment-events-issue-feed)** to see all your deployment events when they arrive, or click a specific deployment to see its APM deployments page. The [APM deployment page](/docs/apm/apm-ui-pages/events/deployments-page-view-impact-your-app-users) lists recent deployments and their impact on your end user and app server's Apdex scores, response times, throughput, and errors. This section will only show up if New Relic has identified applications under the impacted entities that have deployments.
 
-## Impacted entities map 
+## Impacted entities issue map 
 
-![A screenshot ofthe issue entity map view.](./images/issue_entity_map.png "The issue entity map view.")
+![A screenshot of the issue entity map view.](./images/issue_entity_map.png "The issue entity map view.")
```
[the remark this answers is not in the document]
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/use-incident-intelligence.mdx:61]: Good call!

[the remark this answers is not in the document]
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/use-incident-intelligence.mdx:61]: Done!

**Review by kaileyhaynes — APPROVED** [review]
Looks good! More meaningful alt-text :)

**Pull request merged**